tests/www/apply/test_process.py

```diff
@@ -2706,9 +2707,26 @@ def test_accept_updated_birthdate_invalidating_birth_place(self, client, mocker)
         employer = self.company.members.first()
         client.force_login(employer)
 
-        birth_place = Commune.objects.filter(start_date__gt=datetime.date(1901, 12, 1)).first()
-        assert birth_place is not None  # required by test
-
+        birthdate = self.job_seeker.jobseeker_profile.birthdate
+        birth_place = (
+            Commune.objects.filter(
+                # The birthdate must be strictly greater than 1900-01-01, and we’re removing 1 day from start_date.
+                Q(start_date__gt=datetime.date(1900, 1, 2)),
```

Comment:
Je fait pas confiance à mon cerveau aujourd'hui mais pour moi ça devrais être plutôt 1 de ces 2 possibilités :
```suggestion
                Q(start_date__gt=datetime.date(1900, 1, 1)),
```

```suggestion
                Q(start_date__gte=datetime.date(1900, 1, 2)),
```

Reply:
Euh, t'es sûr ?
C'est pas ce que je trouve dans le validateur :
https://github.com/gip-inclusion/les-emplois/blob/f94c6893af26bfd948c5a64929175a76403ca32c/itou/utils/validators.py#L89-L90
Et la DB semble contente :
```
In [6]: JobSeekerProfileFactory(birthdate=datetime.date(1900, 1, 1)).birthdate
Out[6]: datetime.date(1900, 1, 1)
```

Reply:
Non, tu as raison. J’avais retenu ça de mon investigation mais c’est faux.